pom.xml

```diff
@@ -112,6 +112,9 @@
         <opentracing-kafka.version>0.1.13</opentracing-kafka.version>
         <strimzi-oauth.version>0.6.1</strimzi-oauth.version>
         <commons-codec.version>1.13</commons-codec.version>
+        <registry.version>1.3.0.Final</registry.version>
+        <kafka.streams.version>2.5.0</kafka.streams.version>
```

Comment:
We can move to 2.6.0 here I guess, now that we already support Kafka 2.6.0 in 0.20.0 release and master of course.

Reply:
This is tied into Registry's distro.
But I guess if it doesn't break anything, we're fine yo upgrade.
I can check with Registry as well, but afair there we rely on Quarkus' version of Streams ...